name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/wirbank/WirBankPDFExtractorTest.java

```diff
@@ -912,6 +962,41 @@ public void testInterest03()
                         is(Money.of("CHF", Values.Amount.factorize(0.00))));
     }
 
+    @Test
+    public void testInterest04()
+    {
+        Client client = new Client();
+
+        WirBankPDFExtractor extractor = new WirBankPDFExtractor(client);
+
+        List<Exception> errors = new ArrayList<>();
+
+        List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "Zins04_Francais.txt"), errors);
+
+        assertThat(errors, empty());
+        assertThat(results.size(), is(1));
+        new AssertImportActions().check(results, "CHF");
+
+        // check interest transaction
+        AccountTransaction transaction = (AccountTransaction) results.stream().filter(TransactionItem.class::isInstance)
+                        .findFirst().orElseThrow(IllegalArgumentException::new).getSubject();
+
+        assertThat(transaction.getType(), is(AccountTransaction.Type.INTEREST));
+
+        assertThat(transaction.getDateTime(), is(LocalDateTime.parse("2022-06-01T00:00")));
+        assertThat(transaction.getSource(), is("Zins06_Francais.txt"));
+        assertThat(transaction.getNote(), is("Taux d\\'intérêt: 0.10% | Période d\\'intérêt: mai"));
```

Comment:
```suggestion
        assertThat(transaction.getNote(), is("Taux d'intérêt: 0.10% | Période d'intérêt: mai"));
```
We do not need to invalidate any strings or characters here.